README.md

```diff
@@ -283,17 +293,17 @@ Check the official docs on how to [access different content versions](https://ww
 
 ### API
 
-#### useAsyncStoryblok(slug, apiOptions, bridgeOptions)
+#### useAsyncStoryblok(slug, apiParameters, bridgeOptions)
 
 (Recommended Option) Use [`useAsyncData`](https://v3.nuxtjs.org/api/composables/use-async-data/) and [`useState`](https://v3.nuxtjs.org/api/composables/use-state) under the hood for generating SSR or SSG applications.
 
-Check the available [apiOptions](https://github.com/storyblok/storyblok-js-client#class-storyblok) (passed to `storyblok-js-client`) and [bridgeOptions](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-nuxt) (passed to the Storyblok Bridge).
+Check the available [API parameters](https://www.storyblok.com/docs/api/content-delivery/v2#core-resources/stories/retrieve-one-story?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-nuxt) (passed to `storyblok-js-client`) and [bridgeOptions](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-nuxt) (passed to the Storyblok Bridge).
```

Comment:
Agree, the previous link to the section of storyblok-js-client didn't make sense for this function. However, I'm not sure about renaming from apiOptions to apiParameters as I think we refer to it as options in all Clients and SDKs. 🤔

Reply:
But then it will still be confusing since the `apiOptions` in the config of the SDK are not the same, but they are called the same :S

I will revert that for now, but it will be nice to sync on this internally 💜